### DISCUSSION TITLE
Changing Nova tailwind config

### DISCUSSION THREAD

**Discussion by kjoedion** [laravel/nova-issues#4782] [Q&A] [answered]
How would I, for example, change the greys used by Nova from Slate to Neutral?

Normally I would do this by extending the theme in the tailwind config file.

**Comment by davidhemphill** [comment, marked as answer]
You can create a custom asset (`php artisan nova:asset`) that overrides the published CSS variables. For example, your custom asset's CSS file could look something like this:

```css
/* Nova Asset CSS */
:root {
    --colors-gray-100: 255, 0, 0 /** <-- RGB values **/
}
```
